client/src/components/InlinePanel/index.js

```diff
@@ -243,5 +273,6 @@ export class InlinePanel extends ExpandingFormset {
       if (this.opts.onAdd) this.opts.onAdd(formIndex);
       if (this.opts.onInit) this.opts.onInit(formIndex);
     }
+    this.initialFocus(newChildPrefix);
```

Comment:
Let's keep this simple by passing in the selected INNER container as a jQuery node to this function. The current code flow is a bit convoluted and we can assume the exact ID that we want to target instead of going through each node and matching based on an ID.



```suggestion
    this.initialFocus($(`#inline_child_${newChildPrefix}-panel-content`));
```

Note that this 

<img width="1604" alt="Screenshot 2023-09-08 at 8 17 53 am" src="https://github.com/wagtail/wagtail/assets/1396140/958cb6a2-8688-4c51-a98e-22c9e1f9da5d">

<img width="667" alt="Screenshot 2023-09-08 at 8 17 37 am" src="https://github.com/wagtail/wagtail/assets/1396140/772a8ec7-f9ea-4e90-9cb2-d0600436100d">